backend/user-service/models/user-model.js

```diff
@@ -0,0 +1,46 @@
+const pool = require("../psql.js");
+
```

Comment:
Why not we use ORM tool like Prisma (the one I used), Drizzle, or any equivalent tool to help us handle database query so that we don't need to worry about writing them anymore? I think that will help us a lot in speeding up the development speed, and decrease the tests needed.

Reply:
Thanks for the recommendation, I didn't know about these tools, I'll give it a try and update this asap.